src/price_data.py

```diff
@@ -627,15 +633,22 @@ def get_price(
             return decimal.Decimal("1")
 
         db_path = self.get_db_path(platform)
-        tablename = self.get_tablename(coin, reference_coin)
+        if coin < reference_coin:
+            coin_a = coin
+            coin_b = reference_coin
+            inverted = False
+        else:
+            coin_a = reference_coin
+            coin_b = coin
+            inverted = True
+        tablename = self.get_tablename(coin_a, coin_b)
```

Comment:
As this logic is also used in set_price, we should create a function which returns all these... something like

```
def get_XYZ(coin, reference_coin) -> (coin_a, coin_b, tablename, reciprocal):
  if reciprocal := coin > reference_coin:
    ...
  else:
    ....
  tablename = self.get_tablename(coin, reference_coin)

```